docs/tutorial/debugging-vscode.md

```diff
@@ -9,8 +9,8 @@ This guide goes over how to set up VSCode debugging for both your own Electron p
 #### 1. Open an Electron project in VSCode.
 
 ```sh
-$ git clone git@github.com:electron/electron-quick-start.git
-$ code electron-quick-start
+$ git clone git@github.com:electron/minimal-repro.git
+$ code minimal-repro
```

Comment:
I'd suggest using a different project here. We'd have to think about which project would be a good fit. Maybe Fiddle?